__tests__/app.test.js

```diff
@@ -169,6 +169,59 @@ describe('GET /api/articles', () => {
             }
         });
     });
+    test('returns array of article objects which match given topic', () => {
+        return request(app)
+        .get('/api/articles?topic=mitch')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles;
+
+            expect(Array.isArray(articles)).toBe(true);
+
+            expect(articles.length > 0).toBe(true);
+
+            for(article of articles) {
+                expect(article.topic).toBe('mitch');
+            }
+        });
+    });
+    test('returns empty array when given topic that no articles are under', () => {
+        return request(app)
+        .get('/api/articles?topic=AHHHHH')
+        .expect(200)
+        .then((response) => {
+            const articles = response.body.articles;
+
+            expect(articles).toEqual([]);
+        });
+    });
```

Comment:
I like the idea for this, but this should actually throw a 404 error. As "AHHHH" is not a topic that exists, but the idea behind this task is correct. There is a topic "paper" which holds no articles which should return an empty array and this should be tested for.